examples/pipelineruns/volume-output-pipelinerun.yaml

```diff
@@ -0,0 +1,183 @@
+# This example will be using multiple PVCs and will be run against a regional GKE.
+# This means we have to make sure that the PVCs aren't created in different zones,
+# and the only way to do this is to create regional PVCs.
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: regional-disk
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-ssd
+  replication-type: regional-pd
+---
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: volume-resource-1
+spec:
+  type: storage
+  params:
+  - name: type
+    value: volume
+  - name: storageClassName
+    value: regional-disk
+---
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: volume-resource-2
+spec:
+  type: storage
+  params:
```

Comment:
I'm getting an error running this from the webhook:

```
Error from server (InternalError): error when creating "./examples/pipelineruns/volume-output-pipelinerun.yaml": Internal error occurred: admission webhook "webhook.tekton.dev" denied the request: mutation failed: missing field(s): spec.params.size
Error from server (InternalError): error when creating "./examples/pipelineruns/volume-output-pipelinerun.yaml": Internal error occurred: admission webhook "webhook.tekton.dev" denied the request: mutation failed: missing field(s): spec.params.size
```

Reply:
waaaat

Reply:
shoot, makes one wonder how they ever got this to run XD XD XD

im guessing i added that validation _after_ running it 🤦‍♀  oh well XD